nav2_costmap_2d/src/costmap_2d_ros.cpp

```diff
@@ -41,6 +41,7 @@
 #include <memory>
 #include <string>
 #include <vector>
+#include <iostream>
```

Comment:
I don't think this is needed.

Reply:
This was a suggestion from ament's linting tools

Reply:
Oh sorry ,that was another thing, yes I'll remove that, that's from my debug